<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/surface.png);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>登录</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-image: url();</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="set_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>52</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
font: 13pt &quot;STHeiti&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>设定温度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="current_temp_integer">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>152</y>
      <width>67</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
font: italic 36pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="set_temp">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
font: 75 24pt &quot;微软雅黑&quot;;
color: rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string>设定信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up_temp_button">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>420</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>提高目标温度</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>升温</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="down_temp_button">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>降低设置温度</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>降温</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="master_button">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>450</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>开机</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="windspeed_button">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>480</y>
      <width>131</width>
      <height>43</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>降低目标温度</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>风速</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="model_button">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>410</y>
      <width>131</width>
      <height>43</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>模式</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="login_state_label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>52</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
color: rgb(0, 0, 0);
font: 13pt &quot;STHeiti&quot;;</string>
    </property>
    <property name="text">
     <string>室温与风速：</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="speed_group">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>274</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 11pt &quot;STHeiti&quot;;
background-image:url();
background-color: rgb(85, 85, 255, 0%);</string>
    </property>
    <property name="title">
     <string>三种风速选择</string>
    </property>
    <widget class="QRadioButton" name="low_button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>22</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 0, 0);
font: 12pt &quot;STHeiti&quot;;</string>
     </property>
     <property name="text">
      <string>低速风</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="medium_button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>47</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 0, 0);
font: 12pt &quot;STHeiti&quot;;</string>
     </property>
     <property name="text">
      <string>中速风</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="high_button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>72</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 0, 0);
font: 12pt &quot;STHeiti&quot;;</string>
     </property>
     <property name="text">
      <string>高速风</string>
     </property>
    </widget>
    <widget class="QPushButton" name="speed_button">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>100</y>
       <width>50</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-image: url();
background-color: rgb(225, 225, 225);</string>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="login_group">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>300</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="title">
     <string>用户登录</string>
    </property>
    <widget class="QLabel" name="slave_number_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url();
font: 13pt &quot;STHeiti&quot;;</string>
     </property>
     <property name="text">
      <string>从机号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="slave_number_field">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url();
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QLabel" name="password_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url();
font: 13pt &quot;STHeiti&quot;;</string>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_field">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>93</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url();
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginBtn">
     <property name="geometry">
      <rect>
       <x>207</x>
       <y>109</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>next</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="current_temp_fraction">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>181</y>
      <width>28</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
font: italic 20pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="report_group">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>312</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;微软雅黑&quot;;
background-image:url();
background-color: rgb(0, 170, 255,0%);</string>
    </property>
    <property name="title">
     <string>用量统计</string>
    </property>
    <widget class="QLabel" name="consumption_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 13pt &quot;STHeiti&quot;;</string>
     </property>
     <property name="text">
      <string>用量(瓦)：</string>
     </property>
    </widget>
    <widget class="QLabel" name="bill_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 0, 0);
font: 13pt &quot;STHeiti&quot;;</string>
     </property>
     <property name="text">
      <string>金额(元)：</string>
     </property>
    </widget>
    <widget class="QLabel" name="consumption">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bill">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="report">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>报表</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>报表</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>37</width>
      <height>37</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string>close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="min">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>0</y>
      <width>37</width>
      <height>37</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>37</y>
      <width>650</width>
      <height>750</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="model_group">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>286</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;
background-image:url();
background-color: rgb(170, 0, 127, 0%);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="cool_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>选择制冷</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="warm_button">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>选择供暖</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="current_choice">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 0, 0);
font: 12pt &quot;STHeiti&quot;;</string>
     </property>
     <property name="text">
      <string>当前模式：</string>
     </property>
    </widget>
    <zorder>warm_button</zorder>
    <zorder>cool_button</zorder>
    <zorder>current_choice</zorder>
   </widget>
   <widget class="QPushButton" name="login_state">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>5</y>
      <width>19</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="set_label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
font: 13pt &quot;STHeiti&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>设定风速：</string>
    </property>
   </widget>
   <widget class="QLabel" name="set_temp_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
font: 75 24pt &quot;微软雅黑&quot;;
color: rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string>设定信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ccc">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>85</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="work_state_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
color: rgb(0,0,0);
font: 75 24pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>工作状态信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ccc_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>85</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="table" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>441</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="again">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>325</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>175</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 13pt &quot;STHeiti&quot;;</string>
    </property>
    <property name="text">
     <string>实时温度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>650</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 102, 102);
background-color: rgb(102, 102, 102 ,100%);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="work_state">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
color: rgb(0,0,0);
font: 75 24pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>工作状态信息</string>
    </property>
   </widget>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>table</zorder>
   <zorder>current_temp_integer</zorder>
   <zorder>logout</zorder>
   <zorder>up_temp_button</zorder>
   <zorder>down_temp_button</zorder>
   <zorder>windspeed_button</zorder>
   <zorder>model_button</zorder>
   <zorder>current_temp_fraction</zorder>
   <zorder>report</zorder>
   <zorder>report_group</zorder>
   <zorder>master_button</zorder>
   <zorder>close</zorder>
   <zorder>min</zorder>
   <zorder>login_state_label_2</zorder>
   <zorder>set_label</zorder>
   <zorder>set_temp</zorder>
   <zorder>login_state</zorder>
   <zorder>set_label_2</zorder>
   <zorder>set_temp_2</zorder>
   <zorder>ccc</zorder>
   <zorder>work_state_2</zorder>
   <zorder>ccc_2</zorder>
   <zorder>login_group</zorder>
   <zorder>speed_group</zorder>
   <zorder>model_group</zorder>
   <zorder>again</zorder>
   <zorder>label</zorder>
   <zorder>work_state</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
